### DISCUSSION TITLE
Can't get 'People' feature to work

### DISCUSSION THREAD

**Discussion by webserviceXXL** [formbricks/formbricks#2910] [General Requests]
Hi,

I'm really going crazy and hope you can give me a hint. :)

We are currently testing Formbricks because it is by far the most beautiful solution in the FOSS area. However, we are currently failing to clearly identify the users. The documentation is actually easy to understand and clear, but it just won't work. We can neither see the users in the People tab nor in the surveys (always remains 'Anonymous').

I could find a reported bug in the issues nor a topic here in the Discussions. So I just have to believe that it's me.

I build a quick test here: https://azure-possible-noodle.glitch.me/?formbricksDebug=true

Any ideas?

**Comment by webserviceXXL** [comment]
Does someone have an idea what could be wrong or should I create an Issue from this discussion?

The response is working great, but the user is still named "Anonymous" like in the screenshot.

![Screenshot-1 Responses  Onboarding Experience ) Results  Formbricks-Arc-20240726_2006@2x](https://github.com/user-attachments/assets/9fb70f1f-cb4a-4312-9c91-a96273841f3a)

**Comment by mattinannt** [comment]
@webserviceXXL Sorry for the late response; we are currently not actively using Github Discussions and using our Discord server for support instead ( https://formbricks.com/discord ).

You need to explicitly identify the person who is making the response to see them in the `People` view. How are you running your surveys? As Link surveys or in-App surveys via the SDK?
E.g. for link surveys you can identify them with the `userId` url parameter: https://formbricks.com/docs/link-surveys/user-identification

**Reply by wcummings** [reply]
This link is broken

**Reply by jobenjada** [reply]
https://formbricks.com/docs/xm-and-surveys/surveys/website-app-surveys/user-identification

This should work :)
